content/chainguard/chainguard-images/getting-started/getting-started-istio.md

```diff
@@ -0,0 +1,144 @@
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [istioctl](https://istio.io/latest/docs/setup/getting-started/#download)
+
+
+{{< details "What is Wolfi" >}}
+{{< blurb/wolfi >}}
+{{< /details >}}
+
+{{< details "Chainguard Images" >}}
+{{< blurb/images >}}
+{{< /details >}}
+
+## Start up a KinD cluster
+
+First, we'll start up a KinD cluster to install Istio
+```
+kind create cluster
+```
+
+You should see output similar to the following:
+```
+❯ kind create cluster
+Creating cluster "kind" ...
+ ✓ Ensuring node image (kindest/node:v1.27.3) 🖼 
+ ✓ Preparing nodes 📦  
+ ✓ Writing configuration 📜 
+ ✓ Starting control-plane 🕹️ 
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-kind"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-kind
+
+Thanks for using kind! 😊
+```
+
+## Install Istio using Chainguard Images
+
+We will be using the `istioctl` command to install Istio. In order to use the
+Chainguard images, we will need to set these following values:
+- `hub = cgr.dev/chainguard`
+- `tag = latest`
+- `values.pilot.image = istio-pilot`
+- `values.global.proxy.image = istio-proxy`
+- `values.global.proxy_init.image = istio-proxy`
+
+Those translate to the following `istioctl` command:
+```
+istioctl install --set tag=latest --set hub=cgr.dev/chainguard \
+  --set values.pilot.image=istio-pilot \
+  --set values.global.proxy.image=istio-proxy \
+  --set values.global.proxy_init.image=istio-proxy
+```
+
+## Stand up a Gateway and a VirtualService 
+
+To see the Istio installation in action, we will create two Istio resources:
+* An Istio gateway serving the "http://hello.example.com" domain
+* A VirtualService to always reply with "Hello, world!" to requests to the
+  "http://hello.example.com" domain 
+
+Create a YAML file with the following contents to define the Istio resources: 
+```
+cat > example.yaml <<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: sample-gateway
+spec:
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "hello.example.com"
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: sample-virtual-service
+spec:
+  gateways:
+  - sample-gateway
+  hosts:
+  - "hello.example.com"
+  http:
+  - directResponse:
+      status: 200
+      body:
+        string: "Hello, world!\n"
+EOF
+```
+
+Apply the YAML file to the cluster:
+```
+kubectl apply -f example.yaml
+```
+
+Now, in one terminal, start a port-forward to the Istio Ingress Gateway:
+```
+kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80
+```
+
+In another terminal, send a request to the Istio Ingress Gateway:
+```
+curl -H "Host: hello.example.com" localhost:8080
+```
+You should see `Hello, world!` output to the terminal.
+
+## Clean up your KinD cluster 
+
+Once you are done, you can delete your KinD cluster with `kind delete cluster`.
```

Comment:
```suggestion
Once you are done, you can delete your KinD cluster:


kind delete cluster


This will delete the default cluster context, `kind`.

## Advanced Usage

{{< blurb/images-advanced image="Istio" >}}

```

Reply:
This wasn't letting me apply the proper formatting, LMK if you'd like me to share that